CMakeLists.txt

```diff
@@ -66,6 +66,9 @@ set_property(CACHE gRPC_SSL_PROVIDER PROPERTY STRINGS "module" "package")
 set(gRPC_PROTOBUF_PROVIDER "module" CACHE STRING "Provider of protobuf library")
 set_property(CACHE gRPC_PROTOBUF_PROVIDER PROPERTY STRINGS "module" "package")
 
+set(gRPC_PROTOBUF_PROVIDER_TYPE "" CACHE STRING "Algorithm for searching protobuf package")
```

Comment:
Hm. What `MODULE` or `CONFIG` means when `gRPC_PROTOBUF_PROVIDER` is `module`?
Please use `gRPC_PROTOBUF_PACKAGE_TYPE` (as I was write above) instead.